clang/lib/DPCT/DPCT.cpp

```diff
@@ -805,9 +811,12 @@ int runDPCT(int argc, const char **argv) {
 
   ExtraIncPaths = OptParser->getExtraIncPathList();
 
-  // TODO: implement one of this for each source language.
-  CudaPath = getCudaInstallPath(OriginalArgc, argv);
-  DpctDiags() << "Cuda Include Path found: " << CudaPath.getCanonicalPath() << "\n";
+  if (!MigrateCmakeScriptOnly) {
```

Comment:
for readability, will it be better if abstract a function like isCUDAHeaderRequired() {return !MigrateCmakeScriptOnly;}?